app/controllers/claims/support/schools/claims_controller.rb

```diff
@@ -44,7 +44,7 @@ def update
     end
   end
 
-  def submit
+  def add
     Claims::Submit.call(
```

Comment:
Given that the behaviour for `submitting/drafting/adding` a claim is now vastly different for a `SupportUser`, I would suggest creating a separate service for when a `SupportUser` performs this action.

Reply:
This has been something I looked at. It's not for this PR but I will put a refactor PR on our board